pkg/controller/controller_servicebroker.go

```diff
@@ -356,8 +356,8 @@ func (c *controller) reconcileServiceBroker(broker *v1beta1.ServiceBroker) error
 		c.recorder.Event(broker, corev1.EventTypeNormal, successFetchedCatalogReason, successFetchedCatalogMessage)
 
 		// Update metrics with the number of serviceclass and serviceplans from this broker
-		metrics.BrokerServiceClassCount.WithLabelValues(broker.Name).Set(float64(len(payloadServiceClasses)))
-		metrics.BrokerServicePlanCount.WithLabelValues(broker.Name).Set(float64(len(payloadServicePlans)))
+		metrics.BrokerServiceClassCount.WithLabelValues(fmt.Sprintf("%s_%s",broker.Name,broker.Namespace)).Set(float64(len(payloadServiceClasses)))
```

Comment:
I'd prefer to add a second label to those metrics instead of using one label like this.
For example:
```
BrokerServiceClassCount = prometheus.NewGaugeVec(
		prometheus.GaugeOpts{
			Namespace: catalogNamespace,
			Name:      "broker_service_class_count",
			Help:      "Number of services classes by Broker.",
		},
		[]string{"broker", "namespace"},
	)
```

Reply:
then:
```
metrics.BrokerServiceClassCount.WithLabelValues(broker.Name, broker.Namespace).Set(float64(len(payloadServiceClasses)))
```

Reply:
That, looks reasonable. In case of ClusterServiceBrokers we can leave this label empty